### ISSUE TITLE
CI is failing on Windows with `Error: Cannot find module '..'`

### ISSUE THREAD

**Issue by romainmenke** [stylelint/stylelint#7860]
see : https://github.com/nodejs/node/issues/53922, nodejs/node#53902

We might need to pin node 22 to 22.4 or lower if this issue is blocking progress.

**Comment by romainmenke** [comment]
53922 was marked as a duplicate of : https://github.com/nodejs/node/issues/53902

**Comment by romainmenke** [comment]
Almost there, waiting on : https://github.com/actions/node-versions/pull/182

**Comment by ybiquitous** [comment]
Let's wait for a new release of actions/node-versions. We can allow failures in only one CI job for Windows and Node.js 22 for a while.

**Comment by ybiquitous** [comment]
Node.js 22.5.1 is now available on actions/setup-node. The retried CI passed: ✅ 
https://github.com/stylelint/stylelint/actions/runs/10035926924/job/27742175426
